<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmdInputDlg</class>
 <widget class="QDialog" name="cmdInputDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="asciiRd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ASCII</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="hexRd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HEX</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cmdTxt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>461</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
